### PR TITLE
fix(tools): include email when generating JWT token

### PR DESCRIPTION
**Pull request by user512**
ISSUES CLOSED: #152

### CODE REVIEW

File: test/integration/__snapshots__/userFlow.test.js.snap [hunk 1]
```diff
@@ -28,6 +28,14 @@ Array [
 
 exports[`createUser should not create duplicate accounts: no errors 1`] = `undefined`;
 
+exports[`createUser should raise an error without email in header: no email 1`] = `
+Array [
+  [GraphQLError: You must provide a vaild email],
```
Comment (ojongerius) [inline test/integration/__snapshots__/userFlow.test.js.snap]: "vaild" should be valid, if you fix the snapshot you'll fix the typo on the code too.

File: test/integration/__snapshots__/userFlow.test.js.snap [hunk 2]
```diff
@@ -28,6 +28,14 @@ Array [
 
 exports[`createUser should not create duplicate accounts: no errors 1`] = `undefined`;
 
+exports[`createUser should raise an error without email in header: no email 1`] = `
```
Comment (ojongerius) [inline test/integration/__snapshots__/userFlow.test.js.snap]: token, not header.

File: test/integration/__snapshots__/userFlow.test.js.snap [hunk 3]
```diff
@@ -28,6 +28,14 @@ Array [
...
+Array [
+  [GraphQLError: You must provide a vaild email],
+]
+`;
+
+exports[`createUser should raise an error without email in header: null 1`] = `null`;
```
Comment (ojongerius) [inline test/integration/__snapshots__/userFlow.test.js.snap]: token instead of header.

File: test/integration/userFlow.test.js [hunk 4]
```diff
@@ -16,6 +16,7 @@ afterAll(async function afterAllTests() {
 const contextNoToken = global.mockedContextWithOutToken;
 const invalidContext = global.mockedContextWithInValidToken;
 const validContextCharlie = global.mockedContextWithValidTokenForCharlie;
+const noEmailContextMarv = global.mockedContextWithNoEmailTokenForMarv;
```
Comment (ojongerius) [inline test/integration/userFlow.test.js]: could we please use contextNoEmail instead? I think using names in constants is fine if we test a scenario where we validate behaviour with multiple users but this only tests the lack of email in the context.

File: test/integration/userFlow.test.js [hunk 5]
```diff
@@ -105,6 +106,17 @@ describe('createUser', () => {
       .catch(done);
   });
 
+  it('should raise an error without email in header', done => {
```
Comment (ojongerius) [inline test/integration/userFlow.test.js]: context instead of header.

File: test/utils/test-environment.js [hunk 6]
```diff
@@ -54,6 +61,14 @@ class MongoEnvironment extends NodeEnvironment {
       headers: headersWithValidTokenForLola
     };
 
+    const headersWithNoEmailTokenForMarv = {
```
Comment (ojongerius) [inline test/utils/test-environment.js]: remove Marv -see comments about not using a name unless it has a function / communicates an intention.

**Review by ojongerius — CHANGES_REQUESTED** [review]

**Comment by user512** [comment]
Thanks for the feedback, will amend the commit today.

**Review by ojongerius — APPROVED** [review]
LGTM 🎉

**Pull request merged**